### PR TITLE
Fix spelling pda scanner entry progress

### PR DESCRIPTION
**Pull request by Coding-Hen**
Looking through logs and found this small spelling error so cleaned it up. Also added a little more information to the log so its a little more meaningful.

### CODE REVIEW

**Review by Jannify — APPROVED** [review]

File: NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs [hunk 1]
```diff
@@ -24,22 +23,21 @@ public override void Process(PDAEntryProgress packet)
...
-                        Log.Info("PDAEntryProgress Upldate Old:" + entry.unlocked + " New" + packet.Unlocked);
+                        Log.Info($"PDAEntryProgress Update For TechType:{techType} Old:{entry.unlocked} New:{packet.Unlocked}");
                         entry.unlocked = packet.Unlocked;
                     }
                 }
                 else
                 {
-                    Log.Info("PDAEntryProgress New TechType:" + techType + " Unlocked:" + packet.Unlocked);
+                    Log.Info($"PDAEntryProgress New TechType:{techType} Unlocked:{packet.Unlocked}");
                     MethodInfo methodAdd = typeof(PDAScanner).GetMethod("Add", BindingFlags.NonPublic | BindingFlags.Static, null, new Type[] { typeof(TechType), typeof(int) }, null);
```
Comment (dartasen) [inline NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs:39]: We might wanna use the Reflection helper class we have

File: NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs [hunk 2]
```diff
@@ -24,22 +23,21 @@ public override void Process(PDAEntryProgress packet)
             using (packetSender.Suppress<PDAEntryProgress>())
             {
                 TechType techType = packet.TechType.ToUnity();
-                PDAScanner.EntryData entryData = PDAScanner.GetEntryData(techType);
 
                 PDAScanner.Entry entry;
                 if (PDAScanner.GetPartialEntryByKey(techType, out entry))
```
Comment (dartasen) [inline NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs:28]:
```suggestion
                if (PDAScanner.GetPartialEntryByKey(techType, out PDAScanner.Entry entry))
```

File: NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs (hunk 1 above)
Comment (Coding-Hen) [reply NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs:39]: Doesn't look like we are able to do that as c# doesnt allow static types (which PDAScanner is) to be used as type arguments which is what our helper uses.

**Review by dartasen — APPROVED** [review]

File: NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs (hunk 1 above)
Comment (Jannify) [reply NitroxClient/Communication/Packets/Processors/PDAScannerEntryProgressProcessor.cs:38]: But you could put the MethodInfo outside the method as a readonly field. Would improve performance a bit.

**Pull request merged**